pipelines/tekton/validate.go

```diff
@@ -0,0 +1,39 @@
+package tekton
+
+import (
+	"errors"
+	"fmt"
+
+	fn "knative.dev/kn-plugin-func"
+)
+
+var (
+	// ErrRuntimeRequired indicates the required value of Function Runtime was not provided
+	ErrRuntimeRequired = errors.New("runtime is required to build")
+
+	ErrBuilpacksNotSupported = errors.New("additional Buildpacks are not supported for on cluster build")
+)
+
+type ErrRuntimeNotSupported struct {
+	Runtime string
+}
+
+func (e ErrRuntimeNotSupported) Error() string {
```

Comment:
I don't see where this is used anywhere.

Reply:
Line 31

Reply:
🤦